# Lab | Inferential statistics - T-test & P-value
### Instructions
We will have another simple example on two sample t test (pooled- when the variances are equal). But this time this is a one sided t-test


In a packing plant, a machine packs cartons with jars. It is supposed that a new machine will pack faster on the average than the machine currently used. To test that hypothesis, the times it takes each machine to pack ten cartons are recorded. The results, in seconds, are shown in the tables in the file files_for_lab/machine.txt. Assume that there is sufficient evidence to conduct the t test, does the data provide sufficient evidence to show if one machine is better than the other


**Step 1:** Define the null hypothesis - **H0: μ1 ≤ μ2**. (μ1 = mean of time of an old machine, μ2 = mean of time of a new machine)

**Step 2:** Define the alternative hypothesis - **Ha: μ1 > μ2**. 

**Step 3:** This is a **one-tailed test**.

**Step 4:** It is a test between two groups, we will use a **t-test**. 

**Step 5:** Level of significance: **It is 0.05**. 


In [4]:
from scipy.stats import ttest_ind, norm

# create the samples
machi_curr = [42.7, 43.6, 43.8, 43.3, 42.5, 43.5, 43.1, 41.7, 44.0, 44.1]
machi_new = [42.1, 41.0, 41.3, 41.8, 42.4, 42.8, 43.2, 42.3, 41.8, 42.7]

ttest_ind(machi_curr, machi_new)

Ttest_indResult(statistic=3.3972307061176026, pvalue=0.0032111425007745158)

### Conclusion

> statistic = 3.40

> p-value = 0.0032

- From the result, we rejected the H0, meaning

    **A new machine will pack faster on the average than the machine currently used**